347/step3.py

```diff
@@ -0,0 +1,17 @@
+class Solution:
+    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
+        frequency_counter = Counter(nums)
+        max_frequency = max(frequency_counter.values())
+        buckets = [[] for _ in range(max_frequency + 1)]
+
+        for key, frequency in frequency_counter.items():
+            buckets[frequency].append(key)
+
+        top_k = []
+
+        for frequency in range(max_frequency, 0, -1):
```

Comment:
好みかもしれませんが以下のように書くこともできます。
私はこちらのほうが逆順に見てることや初期値と終了条件が分かりやすいので好きです。

```suggestion
        for frequency in reversed(range(1, max_frequency + 1)):
```

Reply:
このような書き方もできるのですね、可読性も上がり、ミスも減りそうな気がしました。勉強になります。